uploadNeuroDB/mass_nii.pl

```diff
@@ -1,5 +1,33 @@
 #!/usr/bin/perl
 
+=pod
+
+=head1 NAME
+
+mass_nii.pl -- Generates NIfTI files based on the MINC files available in the
+LORIS database and inserts them into the parameter_file table.
```

Comment:
C<> for the parameter_file table

Reply:
@MounaSafiHarab Good catch!! Fixed now. Thank you!